infra/github_federated_identity/prod/westeurope/locals.tf

```diff
@@ -0,0 +1,18 @@
+locals {
```

Comment:
isn't this repo supposed to contain only modules?

Reply:
It's true, but it might need to log in to Azure for some automation process.
However, ff preferred we can defer the identity creation to the actual need

Reply:
it's ok